### PR TITLE
Updated wizard armour sets

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request

Fixes wizards internals box being empty. Enchanted (i.e. reskinned combat) gloves now included with either wizard armour purchase. 

Gem-encrusted hardsuit slowdown removed. Battlemage armour is no longer 16 charges, instead is 3 charges and self-recharging like the Syndicate shielded hardsuit. Extra charges addon removed. Both hardsuits given the same base armour values. 

## Why It's Good For The Game

The armour is expensive so it should be better than it currently is. If you're concerned about fastmos the removed slowdown on the gem-crusted harsuit will no longer mean you're a sitting duck to get surrounded and ganked. If your concern is survivability in combat the battlemage armour now better fits it's name. The enchanted gloves will mean wizards are less likely to be stopped by a simple shocked door. 

## Changelog
:cl:
fix: Wizards now have a proper internals box. 
tweak: Gem-encrusted hardsuit now has no slowdown.
tweak: Battlemage armour shield is now self-recharging and has 3 charges, like the Syndicate shielded hardsuit. Armour charges addon removed. 
tweak: Armour values on both hardsuits matched. 
add: Enchanted gloves (reskinned combat gloves) are now included in both wizard armour choices.
/:cl:

Ports:
https://github.com/tgstation/tgstation/pull/49947

### CODE REVIEW

**Comment by ItsTisme** [comment]
Its good to see the battle armor not suck ass, Using help intent or a finger gun already depletes charges so it was stupidly easy to just decharge the shield for its price tag

As a side note maby give the wizards internal box the magic internals tank?

**Comment by VictorVow** [comment]
Magic internals tank is already included if you buy the gem-encrusted hardsuit. Normal wizards start with an extended capacity emergency oxygen tank now.

**Comment by OceanFish1** [comment]
oh my god is that a buff??????????????????????

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -910,6 +910,35 @@
...
+/obj/item/clothing/head/helmet/space/hardsuit/shielded/wizard
+	name = "battlemage helmet"
+	desc = "A suitably impressive helmet.."
+	icon_state = "battlemage"
+	item_state = "battlemage"
+	min_cold_protection_temperature = ARMOR_MIN_TEMP_PROTECT
+	max_heat_protection_temperature = ARMOR_MAX_TEMP_PROTECT
+	armor = list("melee" = 40, "bullet" = 40, "laser" = 40, "energy" = 50, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+	actions_types = null //No inbuilt light
+	resistance_flags = FIRE_PROOF | ACID_PROOF
```
Comment (PowerfulBacon) [inline code/modules/clothing/spacesuits/hardsuit.dm:938]: I think the armour of this should be weaker than the gem-hardsuit considering it has a shield

Comment (heepox) [reply code/modules/clothing/spacesuits/hardsuit.dm:938]: how about instead buff the gem-hardsuits armour values a bit more

Comment (VictorVow) [reply code/modules/clothing/spacesuits/hardsuit.dm:938]: The shield is only 3 shots before it has to recharge so it isn't great in extended close combat, more that it allows a skilled wizard to survive better if they're smart about retreating frequently. If anything I'm more in favour of buffing battlemage armour to be slightly more than gem-hardsuit.

**Comment by PowerfulBacon** [comment]
Stale, reopen when resolved.

**Pull request closed without merging**